cuebot/Dockerfile

```diff
@@ -1,79 +1,24 @@
-FROM centos:7 as base
+FROM gradle:jdk11 as build
 
-# Packages needed at both build and runtime.
-RUN yum install -y \
-      java-1.8.0-openjdk.x86_64 \
-      java-1.8.0-openjdk-devel.x86_64 \
-      libaio \
-      which \
-    && yum clean all
+COPY --chown=gradle:gradle ./cuebot /home/gradle/cuebot/
+COPY --chown=gradle:gradle ./proto /home/gradle/proto/
 
-
-# -----------------
-# BUILD
```

Comment:
This build/test/package convention is used by our [Jenkins setup](https://github.com/AcademySoftwareFoundation/OpenCue/blob/master/jenkins/Jenkinsfile) and matches what the [other components](https://github.com/AcademySoftwareFoundation/OpenCue/blob/master/pycue/Dockerfile#L4) are using -- let's stick with that for now please.

Reply:
No problem, is strange that bleeds into the docker file.  That isn't needed with other systems but I've never used Jenkins.

Reply:
It's not exactly a Jenkins thing, it was done by choice -- by structuring the Dockerfile into stages like this, you can then define a series of Jenkins build steps that have a single `docker build` command associated with them. If you check out our [`Jenkinsfile`](https://github.com/AcademySoftwareFoundation/OpenCue/blob/master/jenkins/Jenkinsfile) this will probably be more clear.

The concept of build steps seems pretty standard within build systems so I would imagine this setup is transferrable to other non-Jenkins build systems.

Reply:
I think this was resolved by CI/CD change right?